source/server/admin/admin_html.cc

```diff
@@ -1,26 +1,59 @@
+#include "envoy/server/admin_handler_factory.h"
+
 #include "source/common/html/utility.h"
 #include "source/server/admin/admin.h"
 #include "source/server/admin/stats_html_render.h"
 
 namespace Envoy {
 namespace Server {
 
-Http::Code AdminImpl::handlerAdminHome(Http::ResponseHeaderMap& response_headers,
-                                       Buffer::Instance& response, AdminStream&) {
-  StatsHtmlRender html(response_headers, response, StatsParams());
-  html.tableBegin(response);
+class AdminHandlerHome : public AdminHandler {
+public:
+  AdminHandlerHome(AdminImpl& admin) : admin_(admin) {}
+  const std::string& prefix() const override {
+    static const std::string prefix = "/";
```

Comment:
no static non-PODs. use constexpr absl::string_view and make the interface match.

On a semantic level, why not leave "/" the way it is? Why go through this more verbose methodology for a built-in?